### ISSUE TITLE
Grey screen after launch on Linux

### ISSUE THREAD

**Issue by lolwat97** [KanColleTool/KanColleTool#107]
Hello, I'm running Arch Linux and have installed the tool from the AUR. When I launch it, I get following messages:
Locale: "en_US"
Couldn't open "/kcsapi/api_start2" ( "/home/lolwat/.cache/KanColleTool/KanColleTool/userdata/kcsapi/api_start2.json" ): 5
Couldn't open "/kcsapi/api_get_member/basic" ( "/home/lolwat/.cache/KanColleTool/KanColleTool/userdata/kcsapi/api_get_member/basic.json" ): 5
Couldn't open "/kcsapi/api_port/port" ( "/home/lolwat/.cache/KanColleTool/KanColleTool/userdata/kcsapi/api_port/port.json" ): 5
Couldn't open "/kcsapi/api_get_member/ndock" ( "/home/lolwat/.cache/KanColleTool/KanColleTool/userdata/kcsapi/api_get_member/ndock.json" ): 5
Couldn't open "/kcsapi/api_get_member/kdock" ( "/home/lolwat/.cache/KanColleTool/KanColleTool/userdata/kcsapi/api_get_member/kdock.json" ): 5
libpng warning: iCCP: known incorrect sRGB profile
Received Translation Data!

and the window stays gray.
How can I fix it?

**Comment by uppfinnarn** [comment]
Sounds like an issue with your Flash Player install. Do you have the correct architecture's `flashplugin` installed?

**Comment by lolwat97** [comment]
Yeah, I think so, yaourt -Qs flash returns
local/flashplayer-standalone 11.2.202.442-1
    Macromedia standalone flash player (nonfree)
extra/flashplugin 11.2.202.442-1
    Adobe Flash Player
multilib/lib32-flashplugin 11.2.202.442-1
    Adobe Flash Player for 32-bit Mozilla-based browsers

**Comment by uppfinnarn** [comment]
Strange… I've used [QtWeb](http://qtweb.net) to diagnose Flash Player-related issues before, as it uses largely the same rendering stack as KCTV, see if you can load some random flash video with that.

Error code 5 is [QFileDevice::OpenError](http://doc.qt.io/qt-5/qfiledevice.html#FileError-enum), which usually means it just hasn't been created yet.

**Comment by lolwat97** [comment]
Well, there is no directory ~/.cache/KanColleTool, so I think that error code means just that.

**Comment by lolwat97** [comment]
QTweb has no trouble loading flash videos.

**Comment by SoftwareGuy** [comment]
Try making the ~/.cache/KanColleTool manually. Maybe permissions are not allowing the viewer to create it automatically?

**Comment by lolwat97** [comment]
@SoftwareGuy No, that didn't resolve the issue. I also tried running the tool as root and also made all the directories down to /home/lolwat/.cache/KanColleTool/KanColleTool/userdata/kcsapi/ and all child folders.

**Comment by mariokamenjak** [comment]
I must say that I had the same issue on Ubuntu 15.04. However updating to Ubuntu 15.10. Resolved the issue for some reason.

I must say that I love your software and I hope you guys have the spare time to continue.

**Comment by nonamethesecond** [comment]
I have the exact same problem and I'm running Arch Linux with the last available version of flash player.

**Comment by nonamethanks** [comment]
Furthermore, after manually saving some of the required jsons (I could not find ndock.json but the others were there just fine) in the required path, the program returns an unknown error when loading them. Example:
Error loading "/kcsapi/api_start2" : 0
Error loading "/kcsapi/api_get_member/basic" : 0

**Comment by liclac** [comment]
The JSONs aren't actually required, but rather, they're cache files. The errors are printed simply to make them easier to find, and after launching the viewer once they should be created by intercepting requests.

**Comment by Francesco149** [comment]
same problem on Gentoo x64:

QtWeb displays swf's fine but kancolletool fails with error 5:
![](http://hnng.moe/f/EtH)

```
loli@memes ~ $ equery l *flash*
 * Searching for *flash* ...
[I-O] [  ] media-plugins/vapoursynth-flash3kyuu_deband-9999:0
[IP-] [  ] www-plugins/adobe-flash-11.2.202.626:0
```

it should be noted that QtWeb uses an older version of qtwebkit than the one required by kancolletool's ebuild. Also, I had to remove the widgets use flag requirement for qtwebkit from the ebuild because the useflag doesn't seem to exist anymore (it's probably enabled by default, the build went through just fine).
